x-pack/test/cloud_security_posture_functional/pages/cis_integrations/cspm/cis_integration_aws.ts

```diff
@@ -124,6 +124,7 @@ export default function (providerContext: FtrProviderContext) {
           (await cisIntegration.getFieldValueInEditPage(DIRECT_ACCESS_KEY_ID_TEST_ID)) ===
             directAccessKeyId
         ).to.be(true);
+        expect(await cisIntegration.doesStringExist('Replace secret access key')).to.not.be(null);
```

Comment:
it is very brittle to rely on specific text and on the generic markup (`span>text`). Can we add a proper test id to this component and rely on it? Otherwise every time the wording change in fleet our tests will break

Reply:
Can we add a test ID to query for the element?

Reply:
so this component is actually a fleet component that I use Lazy import to use in our plugin
the reason why I'm using text is because I thought it might be better to not touch their component

Reply:
I don't see a problem in adding a qa attribute to the component in Fleet, any specific concern you have @animehart ? The fact that another team owns the component makes it even more important - we don't want our tests to break because of the wording changes made by another team. The Fleet team also won't be happy to update our tests when they change smth